.github/workflows/ci_cron_weekly.yml

```diff
@@ -10,13 +10,23 @@ on:
     types:
       - synchronize
       - labeled
+  push:
+    # We want this workflow to always run on release branches as well as
+    # all tags since we want to be really sure we don't introduce
+    # regressions on the release branches, and it's also important to run
+    # this on pre-release and release tags.
+    branches:
+    - 'v*'
+    tags:
+    - '*'
```

Comment:
The exotic archs take a long time. Do you really want to wait around for a few hours between tag and release?

Reply:
We don't technically need to wait for the tag CI to pass to do the release, as as soon as the Azure CI passes on the tag the release is done. However, I think it's still useful to have information on what CI passes on the actual tag even if it's too late to affect the release by then (we normally wait for the CI to pass on the latest branch commit to do the release so there's really no reason the tag would fail).